packages/components/src/LikertScaleLegacy/LikertScaleLegacy.module.scss

```diff
@@ -0,0 +1,369 @@
+}
+
+.likertItem:focus-visible .likertItemFill {
+  outline-offset: 1px;
+  outline: $color-blue-500 $border-focus-ring-border-style
+    $border-focus-ring-border-width;
+}
+
+.reversed .likertItem:focus-visible .likertItemFill {
+  outline-color: $color-blue-300;
+}
+
+.itemContainer {
+  width: 100%;
+  white-space: nowrap;
+  display: inline-block;
+  height: $block-height + 2px;
+  font-size: 0;
+}
+
+.legend {
+  text-align: center;
+  position: absolute;
+  top: calc(-1 * #{$spacing-md});
+  left: 0;
+  right: 0;
+
+  @include respond-to(iphone) {
+    position: initial;
+    top: 0;
+  }
+}
+
+.likertItem {
+  margin-right: 2px;
+  margin-left: 2px;
+
+  &[dir="rtl"],
+  [dir="rtl"] & {
+    &:first-child {
+      margin-left: 2px;
+    }
+
+    &:last-child {
+      margin-right: 2px;
+    }
+  }
+
+  @include fill($block-height);
+
+  font-size: 1px;
+  line-height: 1;
+  padding: 0;
+  overflow-x: visible;
+  display: inline-block;
+  width: 18.5%; // 5 columns
+  position: relative;
+
+  &:first-child {
+    margin-right: 2px;
+    margin-left: 0;
+  }
+
+  &:last-child {
+    margin-left: 2px;
+    margin-right: 0;
+  }
+
+  // Hack to bridge the gaps between items so mouse always hovers on something
+  &::before,
+  &::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: 100%;
+    width: 2px;
+    height: 100%;
+  }
+
+  &::after {
+    right: auto;
+    left: 100%;
+  }
+
+  &.suggested,
+  &.selected {
+    .field1 {
+      background-color: $first;
+
+      &::after {
+        background-color: $first;
+      }
+    }
+
+    .field2 {
+      background-color: $second;
+
+      &::after {
+        background-color: $second;
+      }
+    }
+
+    .field3 {
+      background-color: $third;
+
+      &::after {
+        background-color: $third;
+      }
+    }
+
+    .field4 {
+      background-color: $fourth;
+
+      &::after {
+        background-color: $fourth;
+      }
+    }
+
+    .field5 {
+      background-color: $fifth;
+
+      &::after {
+        background-color: $fifth;
+      }
+    }
+  }
+}
+
+.likertItemFill {
+  transition:
+    background 0.1s,
+    border-color 0.1s;
+  height: $block-height;
+  border: $border-solid-border-width $border-solid-border-style $color-gray-500;
+
+  &:hover {
+    transition:
+      background 0s,
+      border-color 0s;
+  }
+
+  &::after {
+    content: " ";
+    display: block;
+    height: 100%;
+    width: 100%;
+    pointer-events: none;
+  }
+
+  &.pop::after {
+    @include pop;
+  }
+}
+
+.reversed .likertItemFill {
+  border-color: rgba($color-white-rgb, 0.65);
+}
+
+.check {
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin: -10px 0 0 -10px;
+  color: black;
+
+  .likertItem1 & {
+    margin-left: -9px;
+
+    [dir="rtl"] & {
+      margin-left: -11px;
+    }
+  }
+
+  .likertItem5 & {
+    margin-left: -12px;
+
+    [dir="rtl"] & {
+      margin-left: -9px;
+    }
+  }
+}
+
+:where(.likertItem4, .likertItem5) .check {
```

Comment:
Is there any potential issues with `:where` or `.is` not being scoped to a class like `.container :where`? I think the specificity of the `.likertItem` within the `where` selector should be ok, but it still feels a little iffy

Reply:
Good spot, we should change this to be safer